examples/connext_dds/group_coherent_presentation/py/GroupCoherentExample_publisher.py

```diff
@@ -0,0 +1,155 @@
+#
+# (c) 2022 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+#
+# RTI grants Licensee a license to use, modify, compile, and create derivative
+# works of the Software solely for use with RTI products.  The Software is
+# provided "as is", with no warranty of any type, including any warranty for
+# fitness for any purpose. RTI is under no obligation to maintain or support
+# the Software.  RTI shall not be liable for any incidental or consequential
+# damages arising out of the use or inability to use the software.
+#
+import random
+import time
+import argparse
+
+import rti.connextdds as dds
+from GroupCoherentExampleType import AlarmCode, Alarm, HeartRate, Temperature
```

Comment:
When you generate the IDL file, you'll see that it's called GroupCoherentExample.py. The way we map modules from IDL to python is not quite the typical python way, so you'll have to do this:

```
from GroupCoherentExample import *

AlarmCode = GroupCoherentExample.AlarmCode
Alarm = GroupCoherentExample.Alarm
...
```

That's because GroupCoherentExample.py is itself module (derived from the IDL file name), and inside it defines a SimpleNamespace called GroupCoherentExample (the IDL module).

Maybe to keep the example simpler we should modify the IDL (only for Python) to remove the `module GroupCoherentExample` and then just this:

```
from GroupCoherentExample import AlarmCode, Alarm, HeartRate, Temperature
```

Reply:
Hmm, so should I generate the type file with rtiddsgen and then use the GroupCoherentExample module? Or should I  update my current type file (with modules not being used) and keep my current from file import TypeA, TypeB line?

Reply:
I've updated everything else. Let me know how I should proceed with this part. This is how I view it:

Use rtiddsgen to generate GroupCoherentExample.py -> update from import line -> Update publisher and subscriber to use correct scope -> Update README to explain how to generate GroupCoherentExample.py (although it will be included anyway)

or
Update GroupCoherentExampleType to use member_annotation instead of type_annotation and not use the module mapping but it keeps the example simpler

Reply:
Copy the IDL file to the Python directory, and remove the `module GroupCoherentExample` from the file. Then generate code and keep the import statement (it should work as-is).